README.md

```diff
@@ -166,48 +175,87 @@ Example:
 		"resource_id": "/subscriptions/0000/resourceGroups/aztfexport-vmdisk/providers/Microsoft.Compute/virtualMachines/aztfexport-test-test",
 		"resource_type": "azurerm_linux_virtual_machine",
 		"resource_name": "res-3"
-	},
-    ...
+	}
 }
 ```
 
-You can generate the mapping file in all other modes (i.e. `resource`, `resource-group`, `query`) by specifying the `--generate-mapping-file` option when running non-interactively, or press <kbd>s</kbd> when running interactively in the resource list stage. Also, each run of `aztfexport` will generate the resource mapping file for you, to record what resources have been imported.
+If your use cases require pre-export modifications, you are not only welcome but also encouraged to manually construct or edit the mapping file. You could want to modify the mapping file if you:
+- Have many resources in a resource group but only wish to export a select few
+- Wish to rename all your resources in a compliant manner
+- Want to manually discover resources under a given resource type (i.e. networking or compute)
+- And other workflows you might discover!
 
-Of course, you are welcome to manually construct or edit the mapping file. Note that only the object value in the mapping file matters, while the key just plays as an identifier in this mode. 
+Note that only the object value in the mapping file matters, while the key just plays as an identifier in this mode. 
+
+### HCL Only
+
+For any of the aforementioned modes (`resource`, `resource-group`, `query`, `mapping-file`), users can add the `--hcl-only` flag:
+```terminal
+aztfexport [command] --hcl-only [other options] <scope>
+```
+Though `aztfexport` will by default export a state file, the `--hcl-only` flag will result in only the following being generated:
+- Any generated `.tf` HCL files
+- (Planned for 0.10, not true as of 0.9) The mapping file `.json`
+- (Planned for 0.10, not true as of 0.9) Any skipped resources in a `.txt`
+
+This helps for scenarios where you may not need the state or are not sure you want to generate the state. If you then wish to export all of the generated configuration to state, you can rerun the tool and utilize `aztfexport mapping-file` to do so.  
+> 💡 `--hcl-only` must target an empty directory to avoid making unwanted changes to preexisting state during the export stage (thus it will not work with `--append`, but will work with `--overwrite`). Use the `-o` flag to specify an empty directory, or if the directory does not exist, it will be created for you.
```

Comment:
```suggestion
> 💡 `--hcl-only` must target an empty directory to avoid making unwanted changes to preexisting state during the export stage (thus it will not work with `--append`, but will work with `--overwrite`).
```
Is this over wordy?